tests/unit_tests/fixtures/common.py

```diff
@@ -73,3 +76,18 @@ def create_columnar_file(
     df.to_parquet(buffer, index=False)
     buffer.seek(0)
     return FileStorage(stream=buffer, filename=filename)
+
+
+@pytest.fixture
+def admin_user() -> User:
+    role = db.session.query(Role).filter_by(name="Admin").one()
+    user = User(
+        first_name="Alice",
+        last_name="Doe",
+        email="adoe@example.org",
+        username="alice_admin",
+        roles=[role],
+    )
+    db.session.add(user)
+    db.session.flush()
+    yield user
```

Comment:
I tried to use the `admin` user that gets created during test startup, and that worked when running the `key_value_test.py` in isolation, but when running the full test suite the user disappears. So it appears there's a test that deletes it. I didn't have time to try to find the culprit, but at any rate I feel it may be a better idea to not rely on any pre-existing fixtures, hence I suggest using this fixture going forward for new tests needing an admin user.